docs/docs/07-examples/14-wing-analytics.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Wing Cli Analytics
+id: wing-analytics
+keywords: [Analytics, Telemetry]
+---
+
+## Wing Anonymous Analytics
+
+The Wing cli collects anonymous usage data in order to help us improve the cli experience. The intent is to collect data that will help us understand how the cli is being used throughout the community, with hopes that this will help us to build the right features and improve the right areas.
```

Comment:
```suggestion
The Wing CLI collects anonymous usage data in order to help us improve. The intent is to collect data that will help us understand how the CLI is being used throughout the community, with hopes that this will help us to build the right features and improve the right areas.
```